connection/session.feature

```diff
@@ -146,7 +146,7 @@ Feature: Connection Session
       | eve     |
       | frank   |
 
-  @ignore-client-java
+
```

Comment:
https://github.com/graknlabs/client-java/pull/227 fixes these scenarios in client-java.